[VAN-121885] Added diagram (#17)
visier/api-samples: dv-export/python/query-changes/README.md

Diagram of the application flow:
<img src="/assets/images/data-version-query-history.png" alt="Data Version Query History Diagram" width="700"/>

The application performs the following operations:
1. **Load data version changes**: The application uses `DVExportApiClient` to run an export job to retrieve the delta (differences) between two data versions in the `DVManager` class. The application waits for the job to complete. After the job completes successfully, the application downloads a file containing the changes to a specific analytic object (new columns, deleted columns, updated columns).

2. **Select analytic object attributes**: From the downloaded export file, the application selects attributes to include in a list query. These are the attributes that the DV Export API didn't retrieve history for. The application also reduces repeated values.

3. **Load history for analytic objects**: The application uses the Data Query API to fetch changes by filtering by the selected attributes. 

> In `restate` mode, it fetches the full history for each instance.

Reviewer: This sounds a little like a full reload. Maybe something like:
```suggestion
In `restate` mode, it fetches the full change event history for each entity instance that has changed.
```

> In `last` mode, it fetches only the most recent change for each instance.

Reviewer: To aligned with the suggestion above:
```suggestion
In `last` mode, it fetches the last state for each entity instance that has changed.
```
